### PR TITLE
Remove script

### PR DESCRIPTION
**Pull request by jsully1720**
Created : remove tags script
Modified : uuid tagging 
Deleted : Bulk_xml_export

### CODE REVIEW

**Review by ToddBradley — COMMENTED** [review]
I'm confused about why you want people to use both pip and poetry. Why not one or the other?

**Review by brian-l — CHANGES_REQUESTED** [review]
the convention in python is to use snake case not camel case. i don't remember if your other PR had this issue or not. in this PR its mixed case so you should probably make that consistent

**Comment by jsully1720** [comment]
The older scripts use pip, as I go through updating or changing scripts I am bringing them up to python 3.10. That is why there is pip and poetry

**Comment by jsully1720** [comment]
I went into both scripts and changed camel to snake

File: python_examples/remove_uuid_tags/remove_tags.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+# return a dict of UUID : Tags for removal
+def map_tags(wb): 
+
+    # a list to hold companies that don't have any tags
+    # dict to contain UUID:tags 
+    no_tags = []
+    company_tags = {} 
+
+    # define num of columns to check for required columns(2) 
+    numColumns = wb.shape[1]
```
Comment (brian-l) [inline python_examples/remove_uuid_tags/remove_tags.py]: missed one :)

Comment (jsully1720) [reply python_examples/remove_uuid_tags/remove_tags.py]: I think that should do it this time.

**Review by brian-l — APPROVED** [review]

**Pull request merged**